### PR TITLE
Deprecate partner api 

### PR DESCRIPTION
**Pull request by rebekaburnett**

### CODE REVIEW

**Comment by rebekaburnett** [comment]
Need to wait until the migration guide is merged.

File: CHANGELOG.md
Comment (cloveras) [inline CHANGELOG.md:13]: Nice. Requires contacting each user individually, since we are doing it quicker than in the API Lifecycle.

File: CHANGELOG.md
Comment (cloveras) [inline CHANGELOG.md:24]:
```suggestion
* Deprecated the Partner API. The Partner API will be phased out in Q4 2023. Please use the
```

**Review by cloveras — APPROVED** [review]
Looks great!

File: CHANGELOG.md
Comment (rebekaburnett) [inline CHANGELOG.md:22]:
```suggestion
## September 2023
```

**Pull request closed without merging**
